### PR TITLE
Fix wrong text line spacing after switch to Qt6

### PR DESCRIPTION
**Pull request by mike-spa**
Resolves: #22189

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Shouldn't https://github.com/musescore/MuseScore/blob/fdf67262c453dc10f6a1c3147f342f2e3f1c2c50/src/engraving/dom/textbase.cpp#L2003 use `height()` also?

File: src/framework/draw/internal/qfontprovider.cpp [hunk 1]
```diff
@@ -70,6 +70,10 @@ void QFontProvider::insertSubstitution(const String& familyName, const String& s
     QFont::insertSubstitution(familyName, substituteName);
 }
 
+/*******************************************************
+ * CAUTION!
+ * This method may be unreliable. Use height() instead.
+ * *****************************************************/
 double QFontProvider::lineSpacing(const Font& f) const
 {
     return QFontMetricsF(f.toQFont(), &device).lineSpacing();
```
[the remark this answers is not in the document]
Comment (igorkorsukov) [reply src/framework/draw/internal/qfontprovider.cpp:79]:
```
double QFontProvider::lineSpacing(const Font& f) const
{
          // comment here 
          // return QFontMetricsF(f.toQFont(), &device).lineSpacing();
          return QFontMetricsF(f.toQFont(), &device).height();
}
```

[the remark this answers is not in the document]
Comment (mike-spa) [reply src/framework/draw/internal/qfontprovider.cpp]: Good point, thanks! Done now

**Comment by mike-spa** [comment]
After further investigation, I'm afraid the problem is even deeper and this may not be a good solution

**Pull request closed without merging**